src/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+window.onload = () => {
+  const seconds = '--seconds: -' + new Date().getSeconds() + 's';
+
+  const minutes = '--minutes: -' + (
+    new Date().getMinutes() * 60
+    + new Date().getSeconds()
+  ) + 's';
+
+  const hours = '--hours: -' + (
+    new Date().getHours() * 3600
+    + new Date().getMinutes() * 60
+    + new Date().getSeconds()
+  ) + 's';
+
+  document.querySelector('.watch__seconds-second').style = seconds;
+
+  document.querySelector('.watch__minutes-minute').style = minutes;
+
+  document.querySelector('.watch__hours-hour').style = hours;
+};
```

Comment:
This task is not to use javascript to create the clock. It's about "stopwatch using CSS animation".
So your solution doesn't satisfy task requirements.

Reply:
removed